code/game/objects/items/weapons/clown_items.dm

```diff
@@ -68,6 +68,12 @@
 	var/honk_sound = 'sound/items/bikehorn.ogg'
 	var/cooldowntime = 20
 
+/obj/item/weapon/bikehorn/syndie
+	desc = "Guaranteed to make you the subject of a Doctor Doctor joke."
+	attack_verb = list("VIOLENTLY HONKED", "VICIOUSLY MOCKED")
+	origin_tech = "combat=3;syndicate=1"
+	force = 25

```

Comment:
I think this is fine as is but I still like the idea of this horn dealing oxy damage to the target, since it would be even more stealthy and to an observer minus the attack text, there would be no blood. Might be a bit too OP though.